templates/include/telemetry/mainPayload.1.schema.json

```diff
@@ -10,7 +10,11 @@
       "properties": { }
     },
     "childPayloads": {
-      "type": "array"
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": { }
```

Comment:
Why the empty properties here?

Reply:
This seems to be the convention for empty objects in the templates.